platform/applemenu/nbproject/project.xml

```diff
@@ -104,7 +104,9 @@
                     </run-dependency>
                 </dependency>
             </module-dependencies>
-            <public-packages/>
+            <public-packages>
+                <package>org.netbeans.modules.applemenu.spi</package>
```

Comment:
This is an API change and as such it should be documented with `arch.xml`, `apichanges.xml` and addition of this module to list of Javadocs.

Or turn this into `friend-packages`. I don't like usage of friend packages much but in this case, it is probably fine as nobody is likely to provide other implementation anyway.